examples/agentInteractionsDemo/app.js

```diff
@@ -175,9 +175,11 @@ new Vue({
         const usersApi = new platformClient.UsersApi();
         const qualityApi = new platformClient.QualityApi();
         const conversationsApi = new platformClient.ConversationsApi();
+        const externalContactsApi = new platformClient.ExternalContactsApi();
         Vue.prototype.$usersApi = usersApi;
         Vue.prototype.$qualityApi = qualityApi;
         Vue.prototype.$conversationsApi = conversationsApi;
+        Vue.prototype.$externalContactsApi = externalContactsApi;
```

Comment:
Does anyone know if you also need to initialize the Vue.prototype.$externalContactsApi to null at the top of the file?

I dont see any direct refs to any of the $ vars on the Vue instance anyway, so, from a js perspective this seems unnecessary, but, not sure of the vueisms that might need this.